src/shared/modules/Header/Header.tsx

```diff
@@ -76,7 +91,7 @@ const Header = () => {
 			</div>
 			<div className="">
 				<div className={style.authButtons}>
-					{userFromStore.token ? (
+					{userData.id !== '' ? (
```

Comment:
why don't you just write userData.id ? (content) : (content), or check length userData.id.length